app/main/views/index.py

```diff
@@ -38,23 +38,31 @@ def landing(service_id, document_id):
 
     service_contact_info = service['data']['contact_link']
     contact_info_type = assess_contact_type(service_contact_info)
+    metadata = _get_document_metadata(service_id, document_id, key)
 
-    if not _get_document_metadata(service_id, document_id, key):
+    if not metadata:
         return render_template(
             'views/file_unavailable.html',
             service_name=service['data']['name'],
             service_contact_info=service_contact_info,
             contact_info_type=contact_info_type,
         )
 
+    if metadata['verify_email']:
```

Comment:
How confident are you that verify email will be a `boolean` and not accidentally something like `'False'`?